### PR TITLE
Add erlang app from rebar3 template 

### LINKED ISSUE #4: Build system
Split from #2 

---

Use http://www.rebar3.org/ Keep in mind https://github.com/tsoding/tsoding_bot/issues/2#issuecomment-306395482

### PR DESCRIPTION
**Pull request by rexim**
Close #4

### CODE REVIEW

**Comment by rexim** [comment]
@ForNeVeR I know you have more experience with Erlang. Could you please help me with a review? Does PR look ok?

**Comment by rexim** [comment]
@rexim revert that Travis config commit. Travis doesn't support rebar3 yet. https://github.com/tsoding/tsoding_bot/issues/5#issuecomment-306531088 We need to workaround that.

**Comment by rexim** [comment]
The MR is gonna be merged on June 7th after 10pm NOVT.

File: .gitignore [hunk 1]
```diff
@@ -59,5 +59,7 @@ ebin/*.beam
 rel/example_project
 .concrete/DEV_MODE
 .rebar
+rebar.lock
```
Comment (ForNeVeR) [inline .gitignore]: According to the documentation, [`rebar.lock` should not be ignored](https://www.rebar3.org/docs/dependencies#section-lock-files). I suggest to commit this file.

File: src/tsoding_bot.app.src [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  {mod, { tsoding_bot_app, []}},
+  {applications,
+   [kernel,
+    stdlib
+   ]},
+  {env,[]},
+  {modules, []},
+
+  {maintainers, []},
+  {licenses, []},
```
Comment (ForNeVeR) [inline src/tsoding_bot.app.src]: You should probably fill `maintainers` and `licenses` fields to avoid confusion.

**Comment by rexim** [comment]
Thanks for the review Mr @ForNeVeR!

**Pull request merged**